### DISCUSSION TITLE
Docker-compose setup issue

### DISCUSSION THREAD

**Discussion by Raythe** [lloesche/valheim-server-docker#669] [Q&A]
Trying to setup a server in preparation for new update. I've had no issues using docker-compose method for palworld/minecraft or atlassian software but this ones got me stumped.

Running this code from the readme:
`mkdir -p $HOME/valheim-server/config $HOME/valheim-server/data
cd $HOME/valheim-server/
cat > $HOME/valheim-server/valheim.env << EOF
SERVER_NAME=My Server
WORLD_NAME=Dedicated
SERVER_PASS=secret
SERVER_PUBLIC=true
EOF
curl -o $HOME/valheim-server/docker-compose.yaml https://raw.githubusercontent.com/lloesche/valheim-server-docker/main/docker-compose.yaml
docker-compose up`

I get the error
`Failed to load /home/raythe/snap/docker/2915/valheim-server/valheim.env: open /home/raythe/snap/docker/2915/valheim-server/valheim.env: no such file or directory`

This is whats generated running the code I haven't managed to find anything about why it'd be looking into snap/docker/2915 instead of the locations written in the docker file ($HOME/valheim-server). 2915 is an empty folder.
`drwxrwxr-x 2 raythe raythe 4096 Apr 24 04:12 config
drwxrwxr-x 2 raythe raythe 4096 Apr 24 04:12 data
-rw-rw-r-- 1 raythe raythe 386 Apr 24 04:12 docker-compose.yaml
-rw-rw-r-- 1 raythe raythe 82 Apr 24 04:12 valheim.env
`
I'm just a bit lost

**Comment by fizch** [comment]
I was facing a similar issue. I followed the systemd and docker setup instructions, but the error message was the same. I was using the snap installed version of docker. Once I removed that and used apt to install docker (and later docker compose), I was able to get everything up and running.
